frontend/public/components/image-stream.tsx

```diff
@@ -203,24 +235,28 @@ ImageStreamsPage.displayName = 'ImageStreamsListPage';
 
 /*  eslint-disable no-undef, no-unused-vars  */
 type ImageStreamTagsRowProps = {
-  imageStream: any,
-  specTag: any,
-  statusTag: any,
+  imageStream: any;
+  specTag: any;
+  statusTag: any;
+};
+
+export type ImageStreamManipulationHelpProps = {
+  imageStream: any;
```

Comment:
```suggestion
  imageStream: K8sResourceKind;
```

We should change `obj` below to the same.